Composer/packages/client/src/recoilModel/selectors/dialogs.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { DialogInfo } from '@bfc/shared';
+import { selectorFamily } from 'recoil';
+
+import { dialogIdsState, dialogState } from '../atoms';
+
+export const dialogsState = selectorFamily<DialogInfo[], string>({
```

Comment:
the name better to indicate that it's a selectorFamily

Reply:
done